### PR TITLE
Updates registration form

### PR DESCRIPTION
**Pull request by felipefreitag**
- Removes 'country' field
The existing field isn't removed from the model because current users have it on the DB
- Adds terms of service validation according to [Active Record standard](http://guides.rubyonrails.org/active_record_validations.html), section 2.1

### CODE REVIEW

File: db/migrate/20171212192237_change_country_null_on_users.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class ChangeCountryNullOnUsers < ActiveRecord::Migration[5.1]
```
Comment (danielweinmann) [inline db/migrate/20171212192237_change_country_null_on_users.rb]: Why not remove the column instead?

Comment (felipefreitag) [reply db/migrate/20171212192237_change_country_null_on_users.rb]: The production DB has data in that column, I didn't know whether that would be an issue. I can remove it instead

Comment (danielweinmann) [reply db/migrate/20171212192237_change_country_null_on_users.rb]: Is the data relevant? In other words, is there anything other than `Brasil` on that column?

Comment (felipefreitag) [reply db/migrate/20171212192237_change_country_null_on_users.rb]: No, just that.

Comment (danielweinmann) [reply db/migrate/20171212192237_change_country_null_on_users.rb]: Then let's remote the column ;)

**Pull request merged**